packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -88,6 +88,8 @@ export class OdspDocumentService implements IDocumentService {
 
     private _opsCache?: OpsCache;
 
+    private currentConnection?: OdspDocumentDeltaConnection = undefined;
```

Comment:
I don't think we need to initialize it with undefined like other props above?

Reply:
will remove it.